bridge_adaptivity/module/models.py

```diff
@@ -81,8 +81,14 @@ class Sequence(models.Model):
 
     metadata = JSONField(default={}, blank=True)
 
+    # NOTE(yura.braiko) suffix is a hash to make unique user_id for the sequence repetition feature.
```

Comment:
It is not for the sequence repetition it is for collection repetition and availability to use same activity in different collections